hs_core/templates/pages/genericresource.html

```diff
@@ -1618,7 +1642,7 @@ <h4 class="modal-title">Confirm file deletion</h4>
         </div>
 
         <div class="rating col-sm-12 content-block">
-            {% comments_for cm %}
+{#            {% comments_for cm %}#}
```

Comment:
I think it is a mistake to comment this {% comments_for cm %} line, which will disable commenting.